### PR TITLE
Add extension status section.

### PR DESCRIPTION
**Pull request by donmccurdy**

### CODE REVIEW

File: extensions/README.md [hunk 1]
```diff
@@ -8,18 +8,38 @@
...
+consensus forms we may consider the best way to bring a vendor extension into the broader ecosystem: via multi-vendor
+extensions, Khronos extensions, or inclusion in a future version of the glTF specification.
+
+_Draft Khronos (KHR) extensions are not ratified yet. Multi-vendor extensions do not require ratification,
+but may still change before they are complete._
+
+| Extension | Status |
+|-----------|--------|
+| [KHR_techniques_webgl](2.0/Khronos/KHR_techniques_webgl/README.md) | Feature-complete, seeking feedback. |
+| [KHR_lights_punctual](https://github.com/KhronosGroup/glTF/pull/1302) | Feature-complete, seeking feedback. |
```
Comment (scurest) [inline extensions/README.md]: Typo. Link should be to pull/1223.

**Comment by zellski** [comment]
That looks great! Perhaps also settle on some kind of process for updating it as conversation progresses? That may work better if someone knows it's their job. Either one person who's happy to keep up on all the discussions, or else it can be the obligation of whoever created the issue / pull request?

Nit: `KHR_lights` is entirely gone, isn't it? Replaced by *punctual?

**Comment by donmccurdy** [comment]
We're keeping that page updated already, minus the new "highly likely for future development" bit. That isn't well defined, and is really just my subjective opinion from discussions so far... maybe what's needed is a 6-24 month roadmap that is updated on a recurring basis?

Selfishly, this list is also a way of inviting community members with relevant expertise to chime in on things like PBR, for example, where we'd welcome and prioritize feedback for work in this area. I suppose the same could apply for non-extension topics like skinning or performance.

File: extensions/README.md [hunk 2]
```diff
@@ -8,18 +8,38 @@
...
-* KHR_lights *(in progress)*
-
 #### Vendor extensions
 
 * [MSFT_lod](2.0/Vendor/MSFT_lod/README.md)
 * [MSFT_texture_dds](2.0/Vendor/MSFT_texture_dds/README.md)
 * [MSFT_packing_normalRoughnessMetallic](2.0/Vendor/MSFT_packing_normalRoughnessMetallic/README.md)
 * [MSFT_packing_occlusionRoughnessMetallic](2.0/Vendor/MSFT_packing_occlusionRoughnessMetallic/README.md)
 
+### In progress Khronos and multi-vendor extensions
```
[the remark this answers is not in the document]
Comment (pjcozzi) [reply extensions/README.md]: I think it should be `####`.

@donmccurdy is anything else needed before we can merge this?

**Comment by pjcozzi** [comment]
@donmccurdy is this ready to merge?

File: extensions/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (donmccurdy) [reply extensions/README.md]: Done ✅

**Comment by donmccurdy** [comment]
I think this is ready to merge. 👍

**Comment by pjcozzi** [comment]
🎆🎆🎆

**Pull request merged**